examples/official-storybook/config.js

```diff
@@ -15,6 +15,7 @@ addHeadWarning('Dotenv file not loaded', 'dotenv-file-not-loaded');
 setOptions({
   hierarchySeparator: /\/|\./,
   hierarchyRootSeparator: /\|/,
+  theme: themes.dark,
```

Comment:
Will be this a final api of providing the theme?

Reply:
Yeah seems like the right choice for now to me.